core/src/maxicode/MCDecoder.cpp

```diff
@@ -316,8 +316,21 @@ namespace DecodedBitStreamParser
 				result.append(GetMessage(bytes, 1, 77, characterSet, sai));
 				break;
 		}
-		return DecoderResult(std::move(bytes), std::move(result))
+
+		// As converting character set ECIs ourselves and ignoring/skipping non-character ECIs, not using modifiers
+		// that indicate ECI protocol (ISO/IEC 16023:2000 Annexe E Table E1)
+		std::string symbologyIdentifier;
+		if (mode == 4 || mode == 5) {
+			symbologyIdentifier = "]U0";
+		}
+		else if (mode == 2 || mode == 3) {
+			symbologyIdentifier = "]U1";
+		}
+		// No identifier defined for mode 6
+
+        return DecoderResult(std::move(bytes), std::move(result))
```

Comment:
Is there something off with the whitespace here? (Or am I looking at a github artifact?)

Reply:
Yes, messed up the tab on the return - force-pushed [a89b2ca](https://github.com/nu-book/zxing-cpp/commit/a89b2ca0c587c0b2a856350ed7590ada37bc74c6)